scripts/todo.js

```diff
@@ -72,25 +72,25 @@ window.onload = function() {
   // window.indexedDB.open line above
   //it is only implemented in recent browsers
   DBOpenRequest.onupgradeneeded = function(event) {
-    let db = event.target.result;
+    const db = event.target.result;
```

Comment:
We have `let db` near the top, I think it should be:
```suggestion
    db = event.target.result;
```